### ISSUE TITLE
switch to amazon apoly

### ISSUE THREAD

**Issue by beqabeqa473** [tmanderson/ivona-node#14]
hello.

is there any plan to switch to the new aws apoly?

thanks.

**Comment by tmanderson** [comment]
No, there is not. [This is why](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Polly.html). Does that suit your needs?
